index.d.ts

```diff
@@ -3,76 +3,76 @@ declare module 'flagsmith-nodejs' {
      * Initialise the sdk against a particular environment
      */
     export function init(config: {
-        environmentID: string
-        onError?: Function
-        defaultFlags?: string[]
-        api?: string
-    }): void
+        environmentID: string;
```

Comment:
I believe this would need a new entry for the cache option

Reply:
Right! I'll fix that

Reply:
Great, cheers so much for this!